CHANGELOG.md

```diff
@@ -24,3 +26,6 @@ Items without prefix refer to a global change.
 - py: Resolve files in output to actually edit the true file
 
 ## [0.15.0](https://github.com/NNPDF/eko/compare/v0.14.6...v0.15.0) - 2025-03-06
+
+### Added
+- data: Added backwards compatibility system as explained in `src/maintainers.md` ([#448](https://github.com/NNPDF/eko/pull/448))
```

Comment:
- this belongs of course to the "Unreleased" section
- let's be more concise:
  - we didn't add the backward comp. sys. here - that we did #417 
  - instead, we changed for versioning system for unreleased EKOs: from `0.0.0` to `0.0.0-sha` (this is the new format, right?)
  - keep the reference to the PR
  - I would [classify](https://keepachangelog.com/en/1.1.0/) this as "Changed", or what do you think?